site/content/docs/5.2/components/buttons.md

```diff
@@ -6,7 +6,21 @@ group: components
 toc: true
 ---
 
-## Examples
+## Base class
+
+Bootstrap has a base `.btn` class that sets up basic styles such as padding and content alignment. By default, `.btn` controls have a transparent border and background color, and lack any explicit focus and hover styles.
+
+{{< example >}}
+<button type="button" class="btn">Base class</button>
+{{< /example >}}
+
+The `.btn` class is intended to be used in conjunction with our additional predefined button styles, or to serve as a basis for your own custom styles.
+
+{{< callout warning >}}
+If you are using the `.btn` class on its own, remember to at very least define some explicit `:focus` and/or `:focus-visible` styles.
```

Comment:
Not a native English speaker, I'm wondering if this should read "at the very least"?

Reply:
ah, good catch